app/http.php

```diff
@@ -623,7 +625,7 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
                 );
 
                 if (!empty($logs)) {
-                    $error = $th->getMessage() . $logs;
+                    $message = "Failed to execute build command:\n\n" . $logs;
```

Comment:
Im also thinking if its needed to have those prefixes to explain the error. Maybe just $logs would be good 🤔

Can you provide screenshots of failed builds and what they show in logs in Appwrite?

Reply:
Added screenshot

Reply:
Looking at screenshot, I dont think we need our prefix saying something failed 🤔 I think it's fine to just print error, and show red pill saying build failed